README.md

```diff
@@ -1,2 +1,7 @@
 # Dreamers
 Learning fundamentals !
+
+## Students
+Alexandra
```

Comment:
@skoulas the requirements specified that this should be a list
example
- George
- Alexandra